app/navigation/GenericNavigator.scala

```diff
@@ -161,4 +168,20 @@ class GenericNavigator @Inject()() extends Navigator {
         SessionExpiredController.onPageLoad
     }
 
+  private def continueMergeJourney(userAnswers: UserAnswers): Call = {
+    userAnswers.get(TaxYearSelectionPage) match {
+      case Some(taxYearsSelected)  => MergedJourneyController.mergedJourneyContinue(journey="fre", status=getClaimStatus(taxYearsSelected))
+      case _ => SessionExpiredController.onPageLoad
+    }
+  }
+
+  private def getClaimStatus(selectedTaxYears: Seq[TaxYearSelection]): ClaimStatus = {
+    (containsCurrent(selectedTaxYears), containsPrevious(selectedTaxYears)) match {
+      case(true, true) => ClaimCompleteCurrentPrevious
+      case(true,false) => ClaimCompleteCurrent
+      case(false,true) => ClaimCompletePrevious
+      case(_, _) => ClaimUnsuccessful
+    }
+  }
+
```

Comment:
Shouldn't this be in the authenticated navigator

Reply:
Also, if you look at how it decides which confirmation page to use, it ignores the selected years if there are answers for ChangeWhichTaxYearsPage and instead uses the change list